coral-service/src/main/java/com/linkedin/coral/coralservice/utils/CoralUtils.java

```diff
@@ -0,0 +1,87 @@
+package com.linkedin.coral.coralservice.utils;
+
+import com.linkedin.coral.common.HiveMetastoreClient;
+import com.linkedin.coral.common.HiveMscAdapter;
+import com.linkedin.coral.coralservice.metastore.MetastoreProvider;
+import com.linkedin.coral.hive.hive2rel.HiveToRelConverter;
+import com.linkedin.coral.trino.rel2trino.RelToTrinoConverter;
+import com.linkedin.coral.trino.trino2rel.TrinoToRelConverter;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.UUID;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.apache.hadoop.hive.metastore.api.MetaException;
+import org.apache.hadoop.hive.ql.CommandNeedRetryException;
+import org.apache.hadoop.hive.ql.Driver;
+import org.apache.hadoop.hive.ql.metadata.Hive;
+import org.apache.hadoop.hive.ql.metadata.HiveException;
+import org.apache.hadoop.hive.ql.processors.CommandProcessorResponse;
+import org.apache.hadoop.hive.ql.session.SessionState;
+
+/**
+ * Utility class to provide Coral functionality to Coral Service.
+ */
+public class CoralUtils {
```

Comment:
This class seems to be a provider for `HiveMetastoreClient`, `HiveToRelConverter`, `TrinoToRelConverter`. Maybe rename this to CoralProvider and provide `beans` for these objects.  Look into `@Configuration` tags here: https://spring.io/guides/gs/spring-boot/